zerver/webhooks/statuspage/doc.md

```diff
@@ -0,0 +1,22 @@
+Get notifications from Statuspage.io in Zulip whenever any
+company that you subscribed to creates or updates an incident, or updates a component's
+status.
+
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
+
+1. What company needs to do:
+Go to the **subscribers settings** in **notifications** tab and **enable** notifications delivery through **webhook**.
+
```

Comment:
Okay, I am not sure what you mean by this step. For instance is `**enable**` something users click on or is it `**webhook**` that they are supposed to click on! I would recommend being more explicit here and making sure you get capitalisation for the buttons' text right! :)

Reply:
@eeshangarg This setting is something company needs to set up as how they want notifications to be sent to their users (like by email, webhooks or sms)...company must select this if they want to send webhook notifications to their users...

Reply:
Here is the screenshot of that setting : https://github.com/Rishabh570/zulip/blob/statuspage-webhook/static/images/integrations/statuspage/001.png

Reply:
@Rishabh570: I see that, what I meant was that you don't need to bold things like `**enable**` unless they are actual physical section or buttons on the website. And you need to make sure that stuff like `**webhook**` (which is an actual button in the screenshot) is properly capitalised. So, in this case, it would be `**WEBHOOK**`. :)

Reply:
Ooh okay ! I'll change this..